src/platform/packages/shared/kbn-discover-utils/src/utils/sorting/get_sort.ts

```diff
@@ -8,9 +8,9 @@
  */
 
 import type { DataView } from '@kbn/data-views-plugin/common';
-import type { SortOrder } from '@kbn/saved-search-plugin/public';
 import { isPlainObject } from 'lodash';
 
+export type SortOrder = [string, string];
```

Comment:
Leaving `SortOrder` in `saved-search-plugin` created circular TS deps, so I moved it to `kbn-discover-utils` instead and just re-exported from the plugin to avoid updating all usages.